embuilder.py

```diff
@@ -104,6 +104,11 @@
 logger = logging.getLogger(__file__)
 
 def build(src, result_libs, args=[]):
+  # When building ports with the asm.js backend, force WASM to false so that
+  # we don't trigger the downloading of binaryen.
+  if not shared.Settings.WASM_BACKEND and 'WASM=1' not in args:
+    args += ['-s', 'WASM=0']
```

Comment:
this part worries me a little. is it worth adding this in non-test-code just to prevent binaryen from being built in the non-wasm test suite? or is there more to it?

Reply:
Hmm.. I had second thoughts about this part too.  I'll revert it. 

The reason for this change is that I was getting travis failures from the "default." test suite during network issues trying to download binaryen so I tried to remove it completely.. but this maybe does a little far.

Reply:
I see thanks. Cool, lgtm with this part reverted.